tests/baselines/reference/declarationMapsGoToDefinitionSameNameDifferentDirectory.baseline.jsonc

```diff
@@ -0,0 +1,82 @@
+// === /tests/cases/fourslash/server/BaseClass/Source.ts ===
+// class [|Control|]{
+//     constructor(){
+//         return;
+//     }
+//     /** this is a super var */
+//     public myVar: boolean | 'yeah' = true;
+// }
+
+// === /tests/cases/fourslash/server/buttonClass/Source.ts ===
+// // I cannot F12 navigate to Control
+// //                   vvvvvvv
+// class Button extends /*GOTO DEF*/Control {
+//     public myFunction() {
+//         // I cannot F12 navigate to myVar
+//         //              vvvvv
+//         if (typeof this.myVar === 'boolean') {
+//             this.myVar;
+//         } else {
+//             this.myVar.toLocaleUpperCase();
+//         }
+//     }
+// }
+
+// === TextSpan ===
+// === /tests/cases/fourslash/server/buttonClass/Source.ts ===
+// // I cannot F12 navigate to Control
+// //                   vvvvvvv
+// class Button extends /*GOTO DEF*/[|Control|] {
+//     public myFunction() {
+//         // I cannot F12 navigate to myVar
+//         //              vvvvv
+//         if (typeof this.myVar === 'boolean') {
+//             this.myVar;
+//         } else {
+//             this.myVar.toLocaleUpperCase();
+//         }
+//     }
+// }
+
+
+
+// === /tests/cases/fourslash/server/BaseClass/Source.ts ===
+// class Control{
+//     constructor(){
+//         return;
+//     }
+//     /** this is a super var */
+//     public [|myVar|]: boolean | 'yeah' = true;
+// }
+
+// === /tests/cases/fourslash/server/buttonClass/Source.ts ===
+// // I cannot F12 navigate to Control
+// //                   vvvvvvv
+// class Button extends Control {
+//     public myFunction() {
+//         // I cannot F12 navigate to myVar
+//         //              vvvvv
+//         if (typeof this./*GOTO DEF*/myVar === 'boolean') {
+//             this.myVar;
+//         } else {
+//             this.myVar.toLocaleUpperCase();
+//         }
+//     }
+// }
+
+// === TextSpan ===
+// === /tests/cases/fourslash/server/buttonClass/Source.ts ===
+// // I cannot F12 navigate to Control
+// //                   vvvvvvv
+// class Button extends Control {
+//     public myFunction() {
+//         // I cannot F12 navigate to myVar
+//         //              vvvvv
+//         if (typeof this./*GOTO DEF*/[|myVar|] === 'boolean') {
```

Comment:
It seems like the text span doesn’t need its own baseline; it can be combined with the results and disambiguated from definitions by containing the `/*GOTO DEF*/` marker.